docs/reference/ml/trained-models/apis/put-trained-models.asciidoc

```diff
@@ -13,6 +13,12 @@ WARNING: Models created in version 7.8.0 are not backwards compatible
          all nodes must be at least 7.8.0 to use a model stored by
          a 7.8.0 node.
 
+WARNING: As of product version 8.10.0 a new version number has been
+         introduced to track changes to configuration and state in
+         the machine learning plugin. This new version number is
+         decoupled from the product version and will increment
+         independently as necessary.
```

Comment:
I don't think this warning is necessary here. A user putting a new model does not need to know this, and certainly not as a warning.